packages/admin/config/filament.php

```diff
@@ -211,6 +211,7 @@
             'actions' => [
                 'type' => Action::class,
             ],
+            'default_table_records_per_page_select_option' => 10,
```

Comment:
This belongs in the tables config, `tables.pagination.default_records_per_page`

Reply:
Done!